config/configtls/configtls_test.go

```
@@ -923,3 +934,180 @@ func TestCurvePreferences(t *testing.T) {
 		}
 	}
 }
+
+func TestServerConfigValidate(t *testing.T) {
+	tests := []struct {
+		name         string
+		serverConfig ServerConfig
+		errorTxt     string
+	}{
+		{
+			name:         "Valid empty server config",
+			serverConfig: ServerConfig{},
+		},
```

This is the precisely the test we want to return an error according to the tracking issue!

---

Makes sense, I’ll update the test so that an empty server config is expected to fail validation.